docs/book/src/user/concepts.md

```diff
@@ -5,73 +5,74 @@
+
+* __Machine-based__ control planes are the most common type, used by tools like kubeadm and kubespray. Dedicated machines are provisioned, running [static pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for components such as [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver/), [kube-controller-manager](https://kubernetes.io/docs/admin/kube-controller-manager/) and [kube-scheduler](https://kubernetes.io/docs/admin/kube-scheduler/).
 
-* __Machine Based__ based control planes are the most common type deployment model and is used by tools like kubeadm and kubespray. Dedicated machines are provisioned running [*static pods*](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for the control plane components such as  [*kube-apiserver*](https://kubernetes.io/docs/admin/kube-apiserver/), [*kube-controller-manager*](https://kubernetes.io/docs/admin/kube-controller-manager/) and [*kube-scheduler*](https://kubernetes.io/docs/admin/kube-scheduler/).
+* __Pod-based__ deployments require an external hosting cluster. The control plane components are deployed using standard *Deployment* and *StatefulSet* objects and the API is exposed using a *Service*.
 
-* __Pod Based__  deployments require an external hosting cluster, the control plane is deployed using standard *Deployment* and *StatefulSet* objects and then the API exposed using a *Service*.
+* __External__ control planes are offered and controlled by some system other than Cluster API, such as GKE, AKS, EKS, or IKS.
 
-* __External__ control planes are offered and controlled by some system other than Cluster API (e.g., GKE, AKS, EKS, IKS).
+As of v1alpha2, __machine-based__ is the only control plane type that Cluster API supports.
 
-As of v1alpha2 __Machine Based__ is the only supported Cluster API control plane type.
 ## Custom Resource Definitions (CRDs)
 
 ### Machine
 
-A "Machine" is the declarative spec for a Node, as represented in Kubernetes core. If a new Machine object is created, a provider-specific controller will handle provisioning and installing a new host to register as a new Node matching the Machine spec. If the Machine's spec is updated, a provider-specific controller is responsible for updating the Node in-place or replacing the host with a new one matching the updated spec. If a Machine object is deleted, the corresponding Node should have its external resources released by the provider-specific controller, and should be deleted as well.
+A "Machine" is the declarative spec for a Node, as represented in Kubernetes core. If a new Machine object is created, a provider-specific controller will provision and install a new host to register as a new Node matching the Machine spec. If the Machine's spec is updated, the controller updates the Node in-place, or replaces the host with a new one matching the updated spec. If a Machine object is deleted, its host resources and corresponding Node will be deleted by the controller.
```

Comment:
> A "Machine" is the declarative spec for a Node

I think it is more appropriate something like:
A "Machine" is the declarative spec for an infrastructure component hosting a Kubernetes node. e.g  a VM

> the controller updates the Node in-place

AFAIK this is not true

> its host resources

It's underlying infrastructure